python/tests/test_logistic_regression.py

```diff
@@ -484,11 +559,314 @@ def test_lr_fit_multiple_in_single_pass(
+                family="multimonial",
+            )
+
+        assert mlor.getRegParam() == 0.1
+        assert mlor.getElasticNetParam() == 0.2
+
+        mlor.setRegParam(0.15)
+        mlor.setElasticNetParam(0.25)
+        assert mlor.getRegParam() == 0.15
+        assert mlor.getElasticNetParam() == 0.25
+
+        mlor.setRegParam(0.1)
+        mlor.setElasticNetParam(0.2)
+
+        mlor.setFeaturesCol("features")
+        mlor.setLabelCol("label")
+
+        if isinstance(mlor, SparkLogisticRegression):
+            mlor.setWeightCol("weight")
+
+        mlor_model = mlor.fit(mdf)
+
+        mlor_model.setProbabilityCol("newProbability")
+        assert mlor_model.getProbabilityCol() == "newProbability"
+
+        with pytest.raises(
+            Exception,
+            match="Multinomial models contain a matrix of coefficients, use coefficientMatrix instead.",
+        ):
+            mlor_model.coefficients
+
+        with pytest.raises(
+            Exception,
+            match="Multinomial models contain a vector of intercepts, use interceptVector instead.",
+        ):
+            mlor_model.intercept
+
+        assert isinstance(mlor_model.coefficientMatrix, DenseMatrix)
+        coef_mat = mlor_model.coefficientMatrix.toArray()
+
+        if fit_intercept == False:
+            assert isinstance(mlor_model.interceptVector, SparseVector)
+        elif isinstance(mlor_model, SparkLogisticRegressionModel):
+            # Note Spark may return a SparseVector of all zeroes
+            assert isinstance(mlor_model.interceptVector, DenseVector) or isinstance(
+                mlor_model.interceptVector, SparseVector
+            )
+        else:
+            # Note Spark Rapids ML returns a DenseVector of tiny non-zeroes
+            assert isinstance(mlor_model.interceptVector, DenseVector)
+
+        intercept_vec = mlor_model.interceptVector.toArray()
+
+        coef_ground = [
+            [0.96766883, -0.06190176],
+            [-0.06183558, 0.96774077],
+            [-0.96773398, 0.06184808],
+            [0.06187553, -0.96768212],
+        ]
+
+        intercept_ground = [
+            1.78813821e-07,
+            2.82220935e-05,
+            1.44387586e-05,
+            4.82081663e-09,
+        ]
+        assert array_equal(coef_mat, np.array(coef_ground), tolerance)
+        assert array_equal(intercept_vec, intercept_ground, tolerance)
+
+        example = mdf.head()
+        if example:
+            mlor_model.predict(example.features)
+            mlor_model.predictRaw(example.features)
+            mlor_model.predictProbability(example.features)
+
+        if isinstance(mlor_model, SparkLogisticRegressionModel):
+            assert mlor_model.hasSummary
+            mlor_model.evaluate(mdf).accuracy == mlor_model.summary.accuracy
+        else:
+            assert not mlor_model.hasSummary
+            with pytest.raises(RuntimeError, match="No training summary available"):
+                mlor_model.summary
```

Comment:
Were you going to add number of iterations to summary?

Reply:
Seems the summary contains many things besides number of iterations. 
Moved this to the created JIRA ticket 775 for tracking.